src/js/components/Total.js

```diff
@@ -0,0 +1,12 @@
+class Total {
+  constructor({ target, stack }) {
+    this.stack = stack;
+    this.element = document.querySelector(target);
+  }
+
+  render({ stack }) {
+    this.element.innerText = stack.length ? stack.join(' ') : 0;
```

Comment:
stack.join(' ') 에 space가 들어가서 올려주신 계산기 데모로 보면 
예를 들어 30/3= 10의 결과가 나온 상태에서 다시 숫자를 누르면(5를 눌렀다고 가정)
10 5 <- 요렇게 사이에 space가 들어갑니다~ 
수정되면 좋을 것 같습니다 🙂
```suggestion
    this.element.innerText = stack.length ? stack.join('') : 0;
```

Reply:
그러네요 감사합니다 수정해놓을게요